docker-compose.yml

```diff
@@ -23,6 +23,8 @@ services:
       - ALLOWED_HOSTS=["*"]
       - CORS_ORIGIN_WHITELIST=('localhost:3000',)
       - GATHER_CASES_CRON_TIME_SECONDS=1440
+      - FACEBOOK_APP_ID=108193869861396
+      - FACEBOOK_APP_SECRET=6c7e431f5dda1d4075f027fe0b667ee8
```

Comment:
Also, I think `FACEBOOK_APP_ID` is public, so there would be no needs to keep it secret.